Sources/PrimerSDK/Classes/Error Handler/Error.swift

```diff
@@ -334,6 +335,13 @@ enum PrimerError: PrimerErrorProtocol {
                                      bundle: Bundle.primerResources,
                                      value: "Failed to load session, please close and try again.",
                                      comment: "Failed to load session, please close and try again. - Primer error message")
+            
+        case .requestFailed:
+            return NSLocalizedString("primer-error-message-request-failed",
```

Comment:
Is there a way we could link this strings to enums? It would be a lot easier to maintain/ensure consistency/avoid typos if the strings were located in one place.

Reply:
I'm not sure that the localized strings will be auto-generated by Xcode if we provide the identifier, comment and value as a variable. I'll test that and let you know. Still, this is out of scope of this PR.